tests/app/celery/test_tasks.py

```diff
@@ -1607,3 +1608,28 @@ def test_process_returned_letters_list_updates_history_if_notification_is_alread
 
     assert [n.status for n in notifications] == ['returned-letter', 'returned-letter']
     assert all(n.updated_at for n in notifications)
+
+
+def test_send_notify_no_reply(mocker, no_reply_template):
```

Comment:
We could also have a test if the database fail but the notification should be sent.